test/specs/common.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Default identation for json files is 2 spaces. See .editorconfig

Reply:
there is nothing about `json` in the `.editorconfig`